Garret branch

GarretYang opened the pull request:
Added a contact us components

HuskyDanny commented on client/package.json:
```diff
@@ -5,14 +5,15 @@
   "dependencies": {
     "axios": "^0.18.0",
     "bulma": "^0.7.4",
+    "bulma-modal-fx": "^1.1.1",
```
what is this package used for?

GarretYang replied:
It was for animations of the modal, but I did not use it at the end

HuskyDanny commented on client/src/components/contactUs.jsx:
```diff
@@ -0,0 +1,105 @@
+import "./contactUs.css";
+
+class ContactUs extends React.Component {
+
```
```suggestion
You could follow my signup logic here, for more details, please check my login.jsx
```
`<div style={{ padding: "4%" }}>
        <form onSubmit={this.handleSubmit}>
          <div className="field">
            <label className="label">Email</label>
            <div className="control has-icons-left has-icons-right">
              <input
                className="input is-primary"
                type="email"
                placeholder="Your Email"
                required
                onChange={e => this.handleChange(e, "email")}
                value={this.state.email}
              />

              <span className="icon is-small is-left">
                <i className="fas fa-envelope" />
              </span>
              {this.emailDuplicateError()}
            </div>
          </div>`

GarretYang replied:
Ok, do you want me to add it and push the code again?

liuhantao9 commented on client/src/components/navbar.jsx:
```diff
@@ -38,10 +39,15 @@ const Navbar = props => {
               <a className="navbar-link">More</a>
 
-              <div className="navbar-dropdown">
+              <div className="navbar-dropdown buttons">
```
Does this class name change all item in dropdown into buttons?

GarretYang commented:
Added onChange and onSubmit handles to the "contactUs" component
Added input validation to the "contactUs" component

onSubmit does not have backend support yet (cant send emails)
started a new component - user profile
